lib/Github/Api/User.php

```diff
@@ -158,14 +160,27 @@ public function subscriptions($username)
      */
     public function repositories($username, $type = 'owner', $sort = 'full_name', $direction = 'asc')
     {
-        return $this->get('/users/'.rawurlencode($username).'/repos', array(
+        return $this->repos($username, array(
             'type' => $type,
             'sort' => $sort,
             'direction' => $direction
         ));
     }
 
     /**
+     * @param null $username
+     * @param array $options
+     *
+     * @return array list of the user repositories
+     */
+    public function repos($username = null, array $options = array())
+    {
+        $url = $username ? '/users/'.rawurldecode($username).'/repos' : '/user/repos';
```

Comment:
Hm.  This seams like two functions mixed in one method. I'm not 100% convinced. It may be better to "repos()" and "myRepos()" (but with better naming)